docs/language/functions.rst

```diff
@@ -217,13 +217,19 @@ Function modifiers are used to check pre-conditions or post-conditions for a fun
 new modifier must be declared which looks much like a function, but uses the ``modifier``
 keyword rather than ``function``.
 
-.. include:: ../examples/function_modifier.sol
+.. include:: ../examples/substrate/function_modifier.sol
   :code: solidity
 
 The function `foo` can only be run by the owner of the contract, else the ``require()`` in its
 modifier will fail. The special symbol ``_;`` will be replaced by body of the function. In fact,
 if you specify ``_;`` twice, the function will execute twice, which might not be a good idea.
 
+On Solana, ``msg.sender`` does not exist, so the usual way to implement a similar test is using
+an `authority` accounts rather than an owner account.
```

Comment:
I'm starting to think that the code blocks on our documentation should have one tab for each target. This is not something for this PR, though.

Reply:
What do you mean by tab?

Reply:
I think he means [something like this](https://global.discourse-cdn.com/standard10/uploads/gohugo/original/2X/f/f4ff340603de07622145cbcc70e9b8cbf316f646.png) but for our targets. I had a similar idea a while ago, basically the same but putting the documentation text itself inside tabs, wherever there are differences between the targets. But I couldn't figure out how to do in RST.